### ISSUE TITLE
Persistent inability to download/errors downloading several SRA accessions

### ISSUE THREAD

**Issue by ryanpe13002** [ncbi/sra-tools#754]
Tl;dr 5/2052 accessions continually fail during the download; another 6/2052 seem to download fastq files successfully but throw odd error messages and I want to make sure these files are not corrupted

To whom it may concern,
 
I am using the SRA toolkit to download select GTEx FASTQ files from 2,052 samples, and am having consistent trouble downloading 11 specific files (all other files were downloaded successfully with no issues). I am using prefetch to pull the *.sra files and using fastq-dump to pull paired-end FASTQ files from the *.sra file.
 
I am using the attached script “0_download.sh” to download the files (the script simply reads in an SRA accession number and inputs it into the respective commands). **Script**: 
[0_download.sh.zip](https://github.com/ncbi/sra-tools/files/10273727/0_download.sh.zip)
 
For 5 of the accession numbers (SRR612335, SRR809943, SRR1072700, SRR1099905, and SRR1351033), I keep getting an error that reads:
 
`err: data unexpected while reading file within network system module - Cannot KStblHttpFileRead`

I can find no explanation on the internet or anywhere describing what this error means. The error files are attached in “https_failed_download.zip” here: 
[https_failed_download.zip](https://github.com/ncbi/sra-tools/files/10273732/https_failed_download.zip)
 
For other 6 accession numbers (SRR1101643, SRR1446436, SRR599192, SRR1379157, SRR1435643, and SRR1471050), I get errors downloading the SRA vdbcache files, but the remainder of the files _seem_ to download successfully and fastq-dump is able to extract paired end fastq files from the SRA file. HOWEVER, I am concerned that these files are incomplete and want to make sure these errors do not represent an issue prior to use.

Error files are found in this ZIP here: 
[errors_but_download_succeeded.zip](https://github.com/ncbi/sra-tools/files/10273826/errors_but_download_succeeded.zip)
 
Please let me know if there are any things I can do to address these errors/whether the fastq files might be corrupted for the latter 6 errors! Thank you so much for any help or guidance you may be able to provide, and happy holidays! 😊
 
Kindest regards,
Ryan Englander

**Comment by klymenko** [comment]
What toolkit version do you have?
